apps/frontend/src/app/3_organisms/ZeroLocForm/CloseCreditLine.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+import { useEffect } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import { SupportedTokens } from '@sovryn/contracts';
+import {
+  Button,
+  FormGroup,
+  Input,
+  InputSize,
+  Paragraph,
+  ParagraphSize,
+  Select,
+  SimpleTable,
+} from '@sovryn/ui';
+
+import { AssetRenderer } from '../../2_molecules/AssetRenderer/AssetRenderer';
+import { translations } from '../../../locales/i18n';
+import { formatValue } from '../../../utils/math';
+import { Row } from './Row';
+
+type CloseCreditLineProps = {
+  collateralValue: string;
+  creditValue: string;
+  availableBalance: string;
+  onSubmit: () => void;
+};
+
+export const CloseCreditLine: FC<CloseCreditLineProps> = ({
+  collateralValue,
+  creditValue,
+  availableBalance,
+  onSubmit,
+}) => {
+  const { t } = useTranslation();
+  const [error, setError] = useState(false);
+  const availableTokens = [SupportedTokens.dllr, SupportedTokens.zusd];
+  const tokens = availableTokens.map(token => ({
+    value: token,
+    label: token.toUpperCase(),
+  }));
+
+  const [creditToken, setCreditToken] = useState<SupportedTokens>(
+    SupportedTokens.dllr,
+  );
+
+  const collateralValueRenderer = useCallback(
+    (value: number) =>
+      `${formatValue(value, 6)} ${SupportedTokens.rbtc.toUpperCase()}`,
+    [],
+  );
+
+  const insufficientBalance = useMemo(() => {
+    return Number(availableBalance) - Number(creditValue);
+  }, [availableBalance, creditValue]);
+
+  useEffect(() => {
+    Number(availableBalance) < Number(creditValue)
+      ? setError(true)
+      : setError(false);
+  }, [creditValue, availableBalance]);
```

Comment:
having useMemo would be cleaner:

```suggestion
  const hasError = useMemo(() => Number(availableBalance) < Number(creditValue), [creditValue, availableBalance]);
```